### DISCUSSION TITLE
I try to login Vercel but get the message `This user account is blocked.`

### DISCUSSION THREAD

**Discussion by khemmapichpanyana** [vercel/community#3614] [Account Issue] [answered]
### Question

How I can solve this issue ?

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.
